packages/components/RadioGroup/src/RadioButton.tsx

```diff
@@ -96,7 +100,7 @@ export const RadioButton = compose<IRadioButtonType>({
     root: [],
     button: [borderTokens],
     innerCircle: [backgroundColorTokens],
-    content: [foregroundColorTokens, textTokens],
+    content: [foregroundColorTokens, textTokens, [{ source: 'textBorderColor', lookup: getPaletteFromTheme, target: 'borderColor' }]],
```

Comment:
Do you need a width on the border? My guess would be that it defaults to 0, or is there something I'm missing?

Reply:
That's in the settings file